glide.yaml

```diff
@@ -6,3 +6,11 @@ import:
   version: a5b47d31c556af34a302ce5d659e6fea44d90de0
 - package: github.com/gorilla/mux
   version: ^1.6.1
+- package: k8s.io/client-go
+  version: v6.0.0
+- package: k8s.io/apimachinery
+  version: release-1.9
+- package: github.com/prometheus/client_golang
+  version: v0.9.0-pre1
```

Comment:
@krasi-georgiev or @simonpasquier , do you know if the 0.9.0 release for the go client is planned in the short/medium term? If not, maybe we should keep using the previous release.

Reply:
no idea , sorry 
if Simon doesn't know either will ask in the irc channel.